spec/crowbar/client/request/backup/upload_spec.rb

```diff
@@ -0,0 +1,100 @@
+#
+# Copyright 2015, SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require_relative "../../../../spec_helper"
+
+describe "Crowbar::Client::Request::Backup::Upload" do
+  context "a request class upload" do
```

Comment:
how would you best replace `body` with `query` [here](https://github.com/crowbar/crowbar-client/blob/master/spec/support/request_examples.rb#L48) ?